pipeline.py

```diff
@@ -60,8 +60,8 @@
 #
 # Update this each time you make a non-cosmetic change.
 # It will be added to the WARC files and reported to the tracker.
-VERSION = '20230513.01'
-USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36'
```

Comment:
real browser here seems to be using the full `(Windows NT 10.0; Win64; x64)` portion here tho

Reply:
yes, you are right, I checked it on my Windows 10 Laptop. I copied it from https://www.whatismybrowser.com/guides/the-latest-user-agent/chrome